File: leakcanary-sample/src/androidTest/java/com/example/leakcanary/tests/InstrumentationExampleApplication.java
package com.example.leakcanary.tests;

import com.example.leakcanary.ExampleApplication;
import com.squareup.leakcanary.InstrumentationLeakDetector;

public class InstrumentationExampleApplication extends ExampleApplication {

  @Override protected void setupLeakCanary() {
    InstrumentationLeakDetector.instrumentationRefWatcher(this)
        .buildAndInstall();
  }
}

File: leakcanary-sample/src/androidTest/java/com/example/leakcanary/tests/InstrumentationExampleApplication.kt
package com.example.leakcanary.tests

import com.example.leakcanary.ExampleApplication
import com.squareup.leakcanary.InstrumentationLeakDetector

class InstrumentationExampleApplication : ExampleApplication() {

  override fun onCreate() {
    super.onCreate()
    InstrumentationLeakDetector.updateConfig()
  }
}
